### PR TITLE
Update build status in README

### PR DESCRIPTION
**Pull request by norrs**
main delivery branch uses https://github.com/jhannes/logevents/actions/workflows/delivery.yml for uploading coverage and last test results.

### CODE REVIEW

**Comment by norrs** [comment]
I missed updating README when doing https://github.com/jhannes/logevents/pull/78

**Comment by norrs** [comment]
FYI @jhannes

**Comment by jhannes** [comment]
Thank you for the attention to detail. 😊

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Apache 2.0 License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.logevents/logevents/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.logevents/logevents)
 [![Javadoc](https://img.shields.io/badge/javadoc-logevents-blue)](https://jhannes.github.io/logevents/apidocs/)
-[![Build Status](https://github.com/jhannes/logevents/workflows/Java%20CI/badge.svg)](https://github.com/jhannes/logevents/actions/workflows/maven.yml)
```
Comment (norrs) [inline README.md:6]: @jhannes I fucked up with a missing `]`. 

image

Comment (norrs) [reply README.md:6]: @jhannes Since I broke it, here is a follow up pull request: https://github.com/jhannes/logevents/pull/81

**Pull request merged**